frontend/src/features/comparisonSeries/ComparisonSeries.tsx

```diff
@@ -53,6 +57,9 @@ const ComparisonSeries = ({
   const location = useLocation();
 
   const { name: pollName } = useCurrentPoll();
+  const { trackEvent } = Plausible({
+    apiHost: process.env.REACT_APP_WEBSITE_ANALYTICS_URL,
+  });
```

Comment:
Don't you think it would make sense to initialize the Plausible client globally, and export it from a dedicated file?

Even if this initialization has no side-effect (I suppose?), it would make the maintainance easier to configure it from a single place.

Reply:
After some tests on a local VM, it seems that initializing `Plausible` several times does have a side-effect: duplicate event handlers will be defined, leading to multiple "pageviews" events being sent to plausible server.

So we need to be sure this client is created only once, outside of the React lifecycle.

Reply:
I created a custom hook in https://github.com/tournesol-app/tournesol/pull/1340/commits/4487a0e132a3bc39f5e1f2b7f28459c2d8695c0e, I'm going to try if it's enough to prevent multiple initialization.

Reply:
It seems to trigger even more page views.

Reply:
I think I've fixed the major problems with:
- https://github.com/tournesol-app/tournesol/pull/1340/commits/315b7af384d0c9bb34e25cab207c4b25ad7dcbf9
- https://github.com/tournesol-app/tournesol/pull/1340/commits/c858ee1bfad79b75e9d7d1e545278b0078e1a309

I'm trying to see if the page views counter is now reliable : )